zuul.d/projects.yaml

```diff
@@ -49,5 +49,3 @@
       - cifmw-molecule-tempest
       - cifmw-molecule-test_deps
     name: openstack-k8s-operators/ci-framework
-    templates:
```

Comment:
you probably didn't edit the ci/templates/project.yml.j2 when you added the templates support for the project?